region/lowernorfair/east/Lower Norfair Fireflea Room.json

```diff
@@ -586,6 +696,96 @@
       "clearsObstacles": ["A"],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 3],
+      "name": "R-Mode Pause Abuse Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"enemyKill": {"enemies": [["Fune"]]}},
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+            {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]}
+          ]}
+        ]},
+        {"canShineCharge": {"usedTiles": 20, "gentleDownTiles": 2, "openEnd": 1}},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
+        "canRModeSparkInterrupt",
+        {"spikeHits": 1}
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Kill the Fune. Crystal Flash or farm the top fireflea. After the shine charge,",
+        "use the spike pit for a pause abuse interrupt.",
+        "Note: farming more than the top fireflea while still in R-Mode will cause Samus to",
+        "become difficult to see."
+      ]
+    },
+    {
+      "link": [2, 3],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          "canTrickyJump",
+          {"and": [
+            {"enemyKill": {"enemies": [["Fune"]]}},
+            "canCarefulJump"
+          ]},
+          {"enemyKill": {"enemies": [["Fune"], ["Boulder"], ["Fune"]]}},
+          "Ice",
+          {"and": [
+            "ScrewAttack",
+            {"or": [
+              "canCarefulJump",
+              "HiJump",
+              "SpaceJump",
+              {"enemyKill": {"enemies": [["Fune"]]}}
+            ]}
+          ]}
+        ]},
+        {"or": [
```

Comment:
This OR block and the one for climbing the room should be reversed.
You're allowed to take damage climbing the room in this direction.
But then the shinespark out of fireflea pit matters